src/main/java/nl/tudelft/scrumbledore/ScrumbledoreGUI.java

```diff
@@ -28,70 +32,62 @@
-    Label powerupLabel = new Label(Constants.poweruplabelText);
-    Label levelLabel = new Label(Constants.levellabelText);
-
-    // Adding the content for the top bar to the container box.
-    topLabels.getChildren().addAll(scoreLabel, powerupLabel, levelLabel, highLabel);
-    mainPane.setTop(topLabels);
-
-    // Creating the bottom handler box as well as the buttons for it.
-    HBox bottomButtons = new HBox();
-    final Button startstopButton = new Button(Constants.startButtonText);
-    Button settingsButton = new Button(Constants.settingsbuttonText);
-    Button exitButton = new Button(Constants.exitbuttonText);
-
-    // Defining the desired action of the startstop button, which is to stop the game when active,
-    // and restart the game when paused.
-    startstopButton.setOnAction(new EventHandler<ActionEvent>() {
-      public void handle(ActionEvent arg0) {
-        startstopButton.setText(Constants.stopButtonText);
-      }
-    });
+    // Setting the content handler group object, to which objects within the game must be added.
+    BorderPane contentHandler = new BorderPane();
 
-    // Defining the desired action of the settings button, which is to show a settings menu.
-    settingsButton.setOnAction(new EventHandler<ActionEvent>() {
-      public void handle(ActionEvent arg0) {
-        System.out.println("SETTINGS MENU WOULD OPEN NOW");
-      }
-    });
+    // Creating of the scene and assigning this scene to the game stage.
+    Scene mainScene = new Scene(contentHandler);
+    gameStage.setScene(mainScene);
 
-    // Defining the desired action of the exit button, which is to quit when pressed.
-    exitButton.setOnAction(new EventHandler<ActionEvent>() {
-      public void handle(ActionEvent event) {
-        System.exit(0);
+    // Creation of a horizontal box for storing top labels and items to display.
+    HBox topItems = new HBox();
+
+    // Linking the labels needed in the top HBox to their constant referneces.
+    Label scoreLabel = new Label(Constants.SCORELABEL);
+    Label highScoreLabel = new Label(Constants.HISCORELABEL);
+    Label powerUpLabel = new Label(Constants.POWERUPLABEL);
+    Label levelLabel = new Label(Constants.LEVELLABEL);
+
+    // Adding the top labels to the top HBox and to the game display interface.
+    topItems.getChildren().addAll(scoreLabel, powerUpLabel, levelLabel, highScoreLabel);
+    contentHandler.setTop(topItems);
+
+    // Creation of the game display canvas, and adding a graphics context object to allow for
+    // simple, call based refreshing. Canvas is then added to the scene of the window.
+    Canvas gameDisplay = new Canvas(Constants.RESOLUTIONX, Constants.CANVASRESOLUTIONY);
+    GraphicsContext gamePainter = gameDisplay.getGraphicsContext2D();
+    contentHandler.setCenter(gameDisplay);
+
+    // TODO remove debug cells, used to test.
+    for (int i = 0; i < 620; i = i + Constants.BLOCKSIZE) {
+      for (int j = 0; j < 620; j = j + Constants.BLOCKSIZE) {
```

Comment:
Shouldn't these values for j in the fot-loop be replaced with constants, instead of hardcoded?